src/Sarif.Viewer.VisualStudio/FixSuggestedActionsSource.cs

```diff
@@ -55,24 +63,11 @@ public FixSuggestedActionsSource(ITextView textView, ITextBuffer textBuffer, IPe
             }
             else
             {
+                // If this text buffer is not associated with a file, it cannot have any SARIF errors.
                 this.fixableErrors = new List<SarifErrorListItem>().AsReadOnly();
             }
         }
 
-        private static ReadOnlyCollection<SarifErrorListItem> GetFixableErrors(IEnumerable<SarifErrorListItem> errors) =>
```

Comment:
>private [](start = 8, length = 7)

Not deleted or changed, just moved down to the bottom with the rest of the private methods.